README.md

```diff
@@ -26,11 +28,6 @@ tracker here on GitHub for these purposes. Building network simulation
 is out of scope for this project (we might offer a different tool,
 though).
 
-*Note* The screenshots below are somewhat outdated, the current
-status offers quite a bit more. See here for a more up to date 
-tutorial.
-https://github.com/arbor-sim/arbor/wiki/Single-Modelling-with-Arbor's-GUI
```

Comment:
I would still refer to the tutorial/wiki page at some point.

Reply:
Good point.